resources/js/app.js

```diff
@@ -7,16 +7,35 @@
 
 require('./bootstrap');
 
-window.Vue = require('vue');
+// window.Vue = require('vue');
 
 /**
  * Next, we will create a fresh Vue application instance and attach it to
  * the page. Then, you may begin adding components to this application
  * or customize the JavaScript scaffolding to fit your unique needs.
  */
 
-Vue.component('example-component', require('./components/ExampleComponent.vue'));
+// Vue.component('example-component', require('./components/ExampleComponent.vue'));
 
-const app = new Vue({
-    el: '#app'
-});
+// const app = new Vue({
+//     el: '#app'
+// });
```

Comment:
Thank you for commenting this out. We're not using it in the app so it's just increasing the bundle size.

Reply:
Removing the unused javascript! 👍 
![image](https://user-images.githubusercontent.com/1141514/96071806-bd8cd880-0e57-11eb-8b09-2a3cc1701bee.png)